### PR TITLE
Add Pull Request Template for github

### PR DESCRIPTION
**Pull request by fabrice-quadrata**
**Description:**
Add Pull Request Github Template

### CODE REVIEW

File: scripts/deployment/deployMainnet.ts [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+    { initializer: "initialize", kind: "uups", unsafeAllow: ["constructor"] }
+  );
+  console.log("QuadReader is deployed: ", reader.address);
+  await reader.deployed();
+  return reader;
+};
+
+(async () => {
+  const signers = await ethers.getSigners();
+  const timelock = signers[0].address;
```
Comment (huy-quadrata) [inline scripts/deployment/deployMainnet.ts:49]: nit: technically `deployer` and not `timelock`

**Review by huy-quadrata — APPROVED** [review]

File: pull_request_template.md [hunk 2]
```diff
@@ -0,0 +1,10 @@
+### New Feature Submissions:
+
+## Description
+<!--- Describe your changes in detail -->
+
+## Checklist
+1. [ ] Unit Testing?
+2. [ ] Backward Compatible?
```
Comment (teddy-quadrata) [inline pull_request_template.md:8]: Is this different than Upgrade Safe?

**Review by teddy-quadrata — APPROVED** [review]

File: pull_request_template.md [hunk 3]
```diff
@@ -0,0 +1,10 @@
...
+3. [ ] Updated Confluence?
+4. [ ] Updated Gitbook?
```
Comment (daniel-quadrata) [inline pull_request_template.md:10]: Updated JIRA

File: scripts/deployment/deployMainnet.ts [hunk 4]
```diff
@@ -0,0 +1,56 @@
+import { SignerWithAddress } from "@nomiclabs/hardhat-ethers/signers";
+import { Contract } from "ethers";
+
+const { ethers, upgrades } = require("hardhat");
+const { parseEther } = require("ethers/lib/utils");
```
Comment (daniel-quadrata) [inline scripts/deployment/deployMainnet.ts:5]: you can use imports here as well
```ts
import { ethers, upgrades } from "hardhat";
import { parseEther } from "ethers/lib/utils";

File: scripts/deployment/deployMainnet.ts [hunk 5]
```diff
@@ -0,0 +1,56 @@
...
+  });
+  await passport.deployed();
+  console.log(`QuadPassport is deployed: ${passport.address}`);
+  return passport;
+};
+
+export const deployReader = async (
+  governance: SignerWithAddress,
+  passport: SignerWithAddress
+): Promise<Contract> => {
```
Comment (daniel-quadrata) [inline scripts/deployment/deployMainnet.ts:35]: no need to explicitly define the return value type here, typescript should automatically figure that out

File: scripts/deployment/deployMainnet.ts [hunk 6]
```diff
@@ -0,0 +1,56 @@
...
+  return reader;
+};
+
+(async () => {
+  const signers = await ethers.getSigners();
+  const timelock = signers[0].address;
+  const governance = await deployGovernance(timelock);
+  const passport = await deployPassport(
+    governance.address,
+    "ipfs://QmSczMjKWDJBoYSFzPAm3MVFznKcHNnR4EJW23Ng1zQAWu"
```
Comment (daniel-quadrata) [inline scripts/deployment/deployMainnet.ts:53]: should we put this guy in an ENV VAR?

**Review by daniel-quadrata — COMMENTED** [review]
LGTM, minor comments

**Pull request merged**